### ISSUE TITLE
feat(patch): open-links-externally links not showing all browser apps

### ISSUE THREAD

**Issue by natedogith1** [revanced/revanced-patches#1642]
### Application

YouTube

### Issue

The method used to open links externally doesn't show all browsers.  Specifically I have [Better Open With](https://play.google.com/store/apps/details?id=com.aboutmycode.betteropenwith&hl=en_US&gl=US) and [Copy Link to Clipboard](https://play.google.com/store/apps/details?id=nl.robwu.copylink&hl=en_US&gl=US) installed, and these don't show as options to open a link from the youtube app.

### Patch

The standard list of browser apps should be shown.  I assume YouTube is using some non-standard android intent when opening links.

### Motivation

I expect to see the all of the browser options.  The official Gmail app shows the correct list.

Here are the app choices I see in ReVanced YouTube with "Open links in browser" enabled:
![Screenshot_20230213-120802_Android System](https://user-images.githubusercontent.com/1638163/219704181-70996abc-d016-4c76-b4d0-a326279a1639.png)

And here are the app choices I see in Gmail:
![Screenshot_20230213-120955_Android System](https://user-images.githubusercontent.com/1638163/219704326-02859b11-34d7-434e-8ea6-22517b4b9b2b.png)

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by johnconner122** [comment]
It can be achieved if you disable chrome.

**Comment by natedogith1** [comment]
> It can be achieved if you disable chrome.

Why would disabling chrome cause more things to show up in the "Open with" menu?  I tried and now links are opening in Samsung Internet without prompting me which browser to use.

**Comment by natedogith1** [comment]
I just re-patched youtube, and I'm still seeing the issue.  Can this be re-opened, or should I create a new issue?

**Comment by oSumAtrIX** [comment]
I don't know what the issue is, but its not related to ReVanced.

**Comment by natedogith1** [comment]
I remember seeing it a while ago with unmodified YouTube, but since custom tabs were added (what open-links-externally was added to disabled), this issue isn't visible in the unmodified app.  Since I can only get the "Open with" dialog to show up in ReVanced, I figured this would be the place to report it.

I'm not too familiar with how things in android work, but I suspect that YouTube is doing something odd with intents (since there's no issue in Gmail) and I figure ReVanced can probably patch that into whatever the standard way is.
